        <html><head></head><body>
        <h1>Report for python-geoip on toolchain
            <a href="index.html">android</a>
        </h1>
        <p>Return code: 5</p>
        <p>Time to run: 00:00:33</p>
        <p>Log output:</p>
        <code><pre><docker run --rm -v /tuscan_data --volumes-from tuscan_data -v /sources --volumes-from sources -v /var/cache/pacman/pkg --volumes-from pkg_cache_android -v logs:/logs make_package_container --sources-directory /sources --shared-directory /tuscan_data python-geoip
:: Synchronizing package databases...
downloading repo.db...
downloading core.db...
downloading extra.db...
downloading community.db...
Found 2743 packages in cache
===> sudo -u tuscan makepkg --nobuild --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: python-geoip 1.3.2-3 (Sat Nov 14 00:59:42 UTC 2015)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Installing missing dependencies...
resolving dependencies...
looking for conflicting packages...

Packages (4) geoip-database-20151103-1  sqlite-3.9.2-1  geoip-1.6.6-1  python2-2.7.10-2

Total Installed Size:  78.24 MiB

:: Proceed with installation? [Y/n] 
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
checking available disk space...
installing sqlite...
installing python2...
Optional dependencies for python2
    tk: for IDLE
    python2-setuptools
    python2-pip
installing geoip-database...
installing geoip...
==> Retrieving sources...
  -> Downloading GeoIP-1.3.2.tar.gz...
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
  -> Extracting GeoIP-1.3.2.tar.gz with bsdtar
==> Starting prepare()...
==> Sources are ready.
Copied source directory to /sources/python-geoip
===> sudo -u tuscan PATH=/tmp/toolchain/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin CC=arm-linux-androideabi-gcc CXX=arm-linux-androideabi-g++ SYSROOT=/build/android-ndk-r10e/platforms/android-21/arch-arm makepkg --noextract --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: python-geoip 1.3.2-3 (Sat Nov 14 00:59:51 UTC 2015)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Starting build()...
/tmp/toolchain/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'bugtrack_url'
  warnings.warn(msg)
/tmp/toolchain/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'test_suite'
  warnings.warn(msg)
/tmp/toolchain/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'tests_require'
  warnings.warn(msg)
running build
running build_ext
building 'GeoIP' extension
creating build
creating build/temp.linux-x86_64-2.7
arm-linux-androideabi-gcc -DNDEBUG -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/tmp/toolchain/include/python2.7 -c py_GeoIP.c -o build/temp.linux-x86_64-2.7/py_GeoIP.o -fno-strict-aliasing
In file included from /tmp/toolchain/include/python2.7/Python.h:58:0,
                 from py_GeoIP.c:21:
/tmp/toolchain/include/python2.7/pyport.h:1029:2: error: #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?)."
 #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?)."
  ^
py_GeoIP.c:23:19: fatal error: GeoIP.h: No such file or directory
 #include "GeoIP.h"
                   ^
compilation terminated.
error: command 'arm-linux-androideabi-gcc' failed with exit status 1
==> ERROR: A failure occurred in build().
    Aborting...
===> Printing config.logs
===> Finished printing config.logs></pre></code>
        </body></html>
